src/core/ActiveRenderer.cpp

```diff
@@ -172,31 +196,44 @@ void ProjectRenderer::run()
 
 	PerfLogTimer perfLog("Project Render");
 
-	Engine::getSong()->startExport();
-	// Skip first empty buffer.
-	Engine::audioEngine()->nextBuffer();
-
-	m_progress = 0;
-
-	// Now start processing
-	Engine::audioEngine()->startProcessing(false);
+	qDebug("start running");
```

Comment:
std::cerr perhaps? Or what's the best way to output? @JohannesLorenz

Reply:
`qDebug` will be removed

Reply:
Okk then